packages/renderer/src/lib/preferences/docker-compat/PreferencesDockerCompatibilitySocketMappingStatus.svelte

```diff
@@ -0,0 +1,118 @@
+<script lang="ts">
+import { faArrowUpRightFromSquare } from '@fortawesome/free-solid-svg-icons';
+import { Tooltip } from '@podman-desktop/ui-svelte';
+import { onMount } from 'svelte';
+import Fa from 'svelte-fa';
+import { router } from 'tinro';
+
+import Label from '/@/lib/ui/Label.svelte';
+import ProviderInfoCircle from '/@/lib/ui/ProviderInfoCircle.svelte';
+import RefreshButton from '/@/lib/ui/RefreshButton.svelte';
+import type { DockerSocketMappingStatusInfo } from '/@api/docker-compatibility-info';
+
+let isMac = $state(false);
+let isWindows = $state(false);
+let isLinux = $state(false);
+
+let dockerSocketMappingStatusInfo: DockerSocketMappingStatusInfo | undefined = $state(undefined);
+
+let engineType: 'kubernetes' | 'podman' | 'docker' | undefined = $state(undefined);
+
+async function refreshSocketMappingStatus(): Promise<void> {
+  dockerSocketMappingStatusInfo = await window.getSystemDockerSocketMappingStatus();
+
+  if (dockerSocketMappingStatusInfo?.serverInfo?.type === 'podman') {
+    engineType = 'podman';
+  } else if (dockerSocketMappingStatusInfo?.serverInfo?.type === 'docker') {
+    engineType = 'docker';
+  } else {
+    engineType = undefined;
+  }
+}
+
+onMount(async () => {
+  const platform = await window.getOsPlatform();
+
+  isMac = platform === 'darwin';
+  isLinux = platform === 'linux';
+  isWindows = platform === 'win32';
+
+  // ask once to get the result
+  await refreshSocketMappingStatus();
+});
+</script>
+
+<div
+  class="bg-[var(--pd-invert-content-card-bg)] rounded-md m-2 divide-x divide-[var(--pd-content-divider)] flex flex-col lg:flex-row">
+  <div class="flex flex-row grow px-2 py-2 justify-between text-[color:var(--pd-invert-content-card-text)]">
+    <div class="flex flex-col">
+      <div class="flex flex-row items-center text-[color:var(--pd-invert-content-card-text)]">
+        System socket status
+
+        <div class="mx-2">
+          <RefreshButton label="Refresh the status" onclick={refreshSocketMappingStatus} />
+        </div>
+        {#if dockerSocketMappingStatusInfo?.status === 'running' && dockerSocketMappingStatusInfo?.serverInfo}
+          <Label name="{dockerSocketMappingStatusInfo.serverInfo.type} is listening">
+            <ProviderInfoCircle type={engineType} />
+          </Label>
+        {:else if dockerSocketMappingStatusInfo?.status === 'unreachable'}
+          <Label name="socket not reachable">
+            <ProviderInfoCircle type={undefined} />
+          </Label>
+        {/if}
+      </div>
+      <div class="mt-2" role="status" aria-label="description of the status">
```

Comment:
I noticed the 'title' and 'description' were the same font size - add text-sm to match the cards in Preferences?

Reply:
pushed a change

Reply:
Looks good.